test/test_remotes.rb

```diff
@@ -180,4 +180,40 @@ def test_modifies_from_many_threads
       assert_equal(0, remotes.all.reject { |r| r[:host] == host }.size)
     end
   end
+
+  def test_mtime
+    Dir.mktmpdir 'test' do |dir|
+      file = File.join(dir, 'remotes')
+      FileUtils.touch(file)
+      remotes = Zold::Remotes.new(file)
+      sleep 2
+      remotes.add('127.0.0.1')
+      assert(Time.now - remotes.mtime <= 1)
+    end
+  end
+
+  def test_read_mtime_from_file
+    Dir.mktmpdir 'test' do |dir|
+      file = File.join(dir, 'remotes')
+      FileUtils.touch(file)
```

Comment:
@emilianodellacasa I think `mtime` is updated on write anyway, so no need for touch.